### PR TITLE
util: removed Exists function

### PR DESCRIPTION
**Pull request by dgonyeo**
Didn't save a significant amount of code or logic.

### CODE REVIEW

File: lib/begin.go [hunk 1]
```diff
@@ -40,11 +40,13 @@ var (
...
-	if err != nil {
+	_, err = os.Stat(a.ContextPath)
+	switch {
+	case os.IsNotExist(err):
+		break
+	case err != nil:
 		return err
-	}
-	if ex {
+	default:
```
Comment (yifan-gu) [inline lib/begin.go]: Here if  `os.IsNotExist(err) == true`, the error will be returned by `Begin()`

How about just
```go
if err == nil {
    return fmt.Errorf("build already in progress in this working dir")
}
if !os.IsNotExist(err) {
    return err
}
return nil
```

Comment (yifan-gu) [inline lib/begin.go]: nvm.. didn't expand the function ..

File: lib/common.go [hunk 2]
```diff
@@ -73,12 +71,12 @@ func NewACBuild(cwd string, debug bool) *ACBuild {
...
 func (a *ACBuild) lock() error {
-	ex, err := util.Exists(a.ContextPath)
-	if err != nil {
-		return err
-	}
-	if !ex {
+	_, err := os.Stat(a.ContextPath)
+	switch {
+	case os.IsNotExist(err):
 		return fmt.Errorf("build not in progress in this working dir - try \"acbuild begin\"")
```
Comment (yifan-gu) [inline lib/common.go]: `build not in progress` or `no build in progress`?

Comment (dgonyeo) [inline lib/common.go]: I think I like the phrasing in `no build in progress` more.

File: lib/end.go [hunk 3]
```diff
@@ -17,19 +17,17 @@ package lib
...
-	ok, err := util.Exists(a.ContextPath)
-	if err != nil {
-		return err
-	}
-	if !ok {
+	_, err := os.Stat(a.ContextPath)
+	switch {
+	case os.IsNotExist(err):
 		return fmt.Errorf("build not in progress")
+	case err != nil:
```
Comment (yifan-gu) [inline lib/end.go]: can we keep the error message consistent? 
maybe use a variable?
```go
 errorNoBuildInProgress = fmt.Errorf(`"build not in progress in this working dir - try \"acbuild begin\""`)
```

Comment (dgonyeo) [inline lib/end.go]: Done.

**Comment by yifan-gu** [comment]
LGTM

File: lib/begin.go [hunk 4]
```diff
@@ -40,11 +40,13 @@ var (
...
 		return fmt.Errorf("build already in progress in this working dir")
```
Comment (jonboulle) [inline lib/begin.go]: For symmetry, this could be errBuildInProgress

**Pull request merged**